modules/ivc_champva/app/services/ivc_champva/file_uploader.rb

```diff
@@ -2,20 +2,23 @@
 
 module IvcChampva
   class FileUploader
-    def initialize(form_id, metadata, file_paths, insert_db_row = false) # rubocop:disable Style/OptionalBooleanParameter
+    def initialize(form_id, metadata, file_paths, attachment_ids, insert_db_row = false) # rubocop:disable Style/OptionalBooleanParameter
       @form_id = form_id
       @metadata = metadata || {}
       @file_paths = Array(file_paths)
+      @attachment_ids = attachment_ids
```

Comment:
Why couldn't we include this in the metadata itself? Why does it have to be a new attribute?

Reply:
Some forms doesn't require attachments